pytket/qir/conversion/api.py

```diff
@@ -46,6 +48,7 @@ def pytket_to_qir(
     wfh: Optional[wasm.WasmFileHandler] = None,
     int_type: int = 64,
     cut_pytket_register: bool = False,
+    check_input: bool = True,
```

Comment:
Remove argument here (and from docstring).

Reply:
Sorry, I missed that. I will clean this up at the end.

Reply:
Removed in @cqc-melf
[add check function and clean up](https://github.com/CQCL/pytket-qir/pull/147/commits/6a78ce73daff7ad84672e01b40030fd2c83a46cd)
[6a78ce7](https://github.com/CQCL/pytket-qir/pull/147/commits/6a78ce73daff7ad84672e01b40030fd2c83a46cd)